orchagent/portsorch.cpp

```diff
@@ -871,12 +879,12 @@ bool PortsOrch::addVlanMember(Port vlan, Port port)
     attr.value.u16 = vlan.m_vlan_id;
     attrs.push_back(attr);
 
-    attr.id = SAI_VLAN_MEMBER_ATTR_PORT_ID;
+    attr.id = SAI_VLAN_MEMBER_ATTR_BRIDGE_PORT_ID;
     attr.value.oid = port.m_port_id;
```

Comment:
Should not it be "port.m_bridge_port_id" ? Also you need to be sure that LAG is under bridge (probably you need automatically to create it?), because LAG is not bridged after creation.